api/server/router/container/container_routes_test.go

```diff
@@ -228,3 +230,85 @@ func TestEpConfigForNetMode(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleSysctlBC(t *testing.T) {
+	testcases := []struct {
+		name               string
+		networkMode        string
+		sysctls            map[string]string
+		expEpSysctls       []string
+		expSysctls         map[string]string
+		expWarningContains []string
+		expError           string
```

Comment:
There's no test case defining this field. Could you remove it and add another test checking the various errors returned by `handleSysctlBC`?

Reply:
Looks like I got distracted! I've added tests "migration disabled for newer api", "only migrate eth0", and "net name mismatch".